functions/Find-DbaOrphanedFile.ps1

```diff
@@ -0,0 +1,225 @@
+﻿
+Function Find-DbaOrphanedFile
+{
+<#
+.SYNOPSIS 
+Find-DbaOrphanedFile moves/removes orphaned database files

```

Comment:
The reason for that is because it only finds the orphaned files. You could easily pipe the returned values to Remove-Item or Move-Item to execute those actions. I' not sure we're still willing to create the move-dbaorphanedfile and remove-dbaorphanedfile functions.